install_matlab_runtime.yml

```diff
@@ -0,0 +1,50 @@
+- name: Install Matlab Runtime
+  hosts:
+    - localhost
+  gather_facts: false
+  tasks:
+    - name: Update repository cache
+      become: Yes
```

Comment:
Will everybody who is meant to run this script have sudo rights?

Reply:
Yes. This script will be used by the administrators who are going to manage the SRC for the project. So they have sudo rights.